lib/file.js

```diff
@@ -104,8 +115,74 @@ FileCache.prototype.getCachePath = function (key) {
   return path.resolve(path.join(folderPath, key + this.extension))
 }
 
+
+
+/**
+ * enableCacheCleansing - sets up and manages the timeout for cache cleansing
+ *
+ * @returns {void}
+ */
+FileCache.prototype.enableCacheCleansing = function enableCacheCleansing() {
+  var timeoutTrigger = () => {
+    this.cleanseTimeout = setTimeout(() => {
+      this.cleanseEnabled && this.cleanseCache(this.directory, timeoutTrigger)
+    }, this.cleanseInterval * 1000)
+  }
+
+  timeoutTrigger()
+}
+
 /**
+ * disableCacheCleansing - stops and cleans up the timeout for cache cleansing
  *
+ * @returns {void}
+ */
+FileCache.prototype.disableCacheCleansing = function enableCacheCleansing() {
```

Comment:
should this be 

`FileCache.prototype.disableCacheCleansing = function disableCacheCleansing() {`

?